hi-ml-azure/src/health_azure/datasets.py

```diff
@@ -29,8 +29,9 @@ def get_datastore(workspace: Workspace, datastore_name: str) -> Datastore:
     if not datastore_name:
         if len(existing_stores) == 1:
```

Comment:
I think it's worth adding a comment here that it is possible to have default datastore set, hence why we perform this check first